### PR TITLE
remove the unused testimony tokens & none used for reporting

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### Problem Statement
Remove 3 tokens from Robottelo which are not useful or not being used anywhere in reporting:

TestType
Upstream
CaseLevel

### Solution
removed these three token from the test docstring, This will need some testing against Polarian staging host.

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
"trigger": "test-robottelo"

File: tests/foreman/api/test_contentview.py [hunk 1]
```diff
@@ -188,9 +177,7 @@ def test_positive_add_custom_content(self, module_product, module_org, module_ta
 
         :expectedresults: Custom content assigned and present in content view
 
-        :CaseLevel: Integration
-
-        :CaseImportance: Critical
+                :CaseImportance: Critical
```
Comment (JacobCallahan) [inline tests/foreman/api/test_contentview.py:180]: It looks like there are instances where additional indentation was added. I won't comment on each instance in this PR

File: tests/foreman/api/test_hostgroup.py [hunk 2]
```diff
@@ -649,9 +631,7 @@ def test_positive_create_with_group_parameters(self, module_org, module_target_s
 
         :customerscenario: true
 
-        :CaseLevel: Integration
-
-        :BZ: 1710853
+                :BZ: 1710853
```
Comment (JacobCallahan) [inline tests/foreman/api/test_hostgroup.py:634]: Interestingly, this field was also extra-indented (like the others).

File: tests/foreman/api/test_repository.py [hunk 3]
```diff
@@ -1828,8 +1802,7 @@ def test_negative_synchronize_private_registry_wrong_password(self, repo_options
 
         :BZ: 1475121, 1580510
 
-        :CaseLevel: Integration
-        """
+                """
```
Comment (JacobCallahan) [inline tests/foreman/api/test_repository.py:1805]: extra indentation for some quotes as well

File: tests/foreman/api/test_repository.py [hunk 4]
```diff
@@ -2175,8 +2146,7 @@ def test_negative_synchronize_docker_repo_with_invalid_tags(self, repo_options,
 #
 #         :expectedresults: Synced repo should fetch the data successfully.
 #
-#         :CaseLevel: Integration
-#
+#         #
```
Comment (JacobCallahan) [inline tests/foreman/api/test_repository.py:2149]: This can likely just be removed.
```suggestion
```

File: tests/foreman/cli/test_errata.py [hunk 5]
```diff
@@ -492,6 +485,62 @@ def test_positive_install_by_host_collection_and_org(
 
 @pytest.mark.tier3
 def test_negative_install_erratum_on_host_collection(
+    module_entitlement_manifest_org, host_collection, errata_hosts, target_sat
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_errata.py:488]: There are a number of apparent test additions in this file, is this a history conflict?

File: tests/foreman/ui/test_repository.py [hunk 6]
```diff
@@ -648,8 +624,7 @@ def test_positive_upstream_with_credentials(session, module_prod):
 #
 #     :expectedresults: All expected CRUD actions finished successfully
 #
-#     :CaseLevel: Integration
-#
+#     #
```
Comment (JacobCallahan) [inline tests/foreman/ui/test_repository.py:627]:
```suggestion
```

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

File: tests/foreman/cli/test_errata.py (hunk 5 above)
Comment (omkarkhatavkar) [reply tests/foreman/cli/test_errata.py]: some conflicts are resolved now. rebase with master

File: tests/foreman/api/test_hostgroup.py (hunk 2 above)
Comment (omkarkhatavkar) [reply tests/foreman/api/test_hostgroup.py]: yeah that was really painful

File: tests/foreman/virtwho/ui/test_esx.py [hunk 7]
```diff
@@ -279,9 +262,7 @@ def test_positive_virtwho_roles(self, org_session):
         :expectedresults:
             'Virt-who Manager', 'Virt-who Reporter', 'Virt-who Viewer' existing
 
-        :CaseLevel: Integration
-
-        :CaseImportance: Low
+                :CaseImportance: Low
```
Comment (JacobCallahan) [inline tests/foreman/virtwho/ui/test_esx.py:265]: extra indentation here and another below in this file

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
Getting very close!

File: tests/foreman/api/test_repository.py [hunk 8]
```diff
@@ -2278,9 +2247,7 @@ class TestSRPMRepositoryIgnoreContent:
     In particular sync of duplicate SRPMs would fail when using the flag
     ``ignorable_content``.
 
-    :CaseLevel: Integration
-
-    :CaseComponent: Pulp
+        :CaseComponent: Pulp
```
[the remark this answers is not in the document]
Comment (omkarkhatavkar) [reply tests/foreman/api/test_repository.py]: I am not sure how you found it, I had to write an AST program to find the places. I feel it should be resolved in all places.

**Comment by omkarkhatavkar** [comment]
image

**Review by JacobCallahan — APPROVED** [review]
ACK, well done!!

**Review by jyejare — APPROVED** [review]

**Comment by jyejare** [comment]
trigger: test-robottelo

**Comment by jyejare** [comment]
trigger: test-robottelo

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Comment by jyejare** [comment]
trigger: test-robottelo

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Pull request merged**